cmd/podman/system/connection/list.go

```diff
@@ -118,19 +118,22 @@ func inspect(cmd *cobra.Command, args []string) error {
 		rpt, err = rpt.Parse(report.OriginUser, format)
 	} else {
 		rpt, err = rpt.Parse(report.OriginPodman,
-			"{{range .}}{{.Name}}\t{{.URI}}\t{{.Identity}}\t{{.Default}}\t{{.ReadWrite}}\n{{end -}}")
+			"{{range .}}{{.Name}}\t{{.URI}}\t{{.Identity}}\t{{.TLSCAFile}}\t{{.TLSCertFile}}\t{{.TLSKeyFile}}\t{{.Default}}\t{{.ReadWrite}}\n{{end -}}")
 	}
 	if err != nil {
 		return err
 	}
 
 	if rpt.RenderHeaders {
 		err = rpt.Execute([]map[string]string{{
-			"Default":   "Default",
-			"Identity":  "Identity",
-			"Name":      "Name",
-			"URI":       "URI",
-			"ReadWrite": "ReadWrite",
+			"Default":     "Default",
+			"Identity":    "Identity",
+			"TLSCAFile":   "TLSCAFile",
+			"TLSCertFile": "TLSCertFile",
+			"TLSKeyFile":  "TLSKeyFile",
```

Comment:
ditto

Reply:
Also, I'm not certain I understand how the second is a breaking change, those are just the list of fields *available* for use, not *required* to be used, in a format string. If those are not there, a user wouldn't even be able to `--format` to see them.